beacon-chain/blockchain/process_block.go

```diff
@@ -555,6 +559,25 @@ func (s *Service) handleBlockAttestations(ctx context.Context, blk interfaces.Be
 	return nil
 }
 
+func (s *Service) handleBlockBLSToExecChanges(ctx context.Context, blk interfaces.BeaconBlock) error {
+	if blk.Version() < version.Capella {
+		return nil
+	}
+	changes, err := blk.Body().BLSToExecutionChanges()
+	if err != nil {
+		return errors.Wrap(err, "could not get BLSToExecutionChanges")
+	}
+	for _, change := range changes {
+		if ctx.Err() != nil {
+			return ctx.Err()
+		}
```

Comment:
I think we can remove this. This is a fast loop and should be far lesser than @kasey 's suggestion ~20ms